lib/mtg_cli/data_formatter.rb

```diff
@@ -0,0 +1,32 @@
+module MtgCli
+  module DataFormatter
+    module_function
+
+    def reformat(json_data)
+      reformatted_cards = {}
+      json_data.each do |set, data|
+        cards = parse_set(data)
+        reformatted_cards.merge!(parse_set(data)) do |key, old_card, new_card|
+          old_card['setName'].push(new_card['setName'].first)
+          old_card['setCode'].push(new_card['setCode'].first)
+          old_card
+        end
+      end
+      reformatted_cards.sort_by { |name, data| name }.to_h
+    end
+
+    def parse_set(set_hash)
```

Comment:
This is a pretty big method that's hard to parse... is there any way you could split it up into smaller sub methods?